main.html

```diff
@@ -13,116 +13,193 @@
+            <li><span class="bs">bootstrap</span></li>
+            <li><span class="ruby">Ruby</span></li>
+          </ul>
+          <button type="button" name="See Project">See Project</button>
+        </section>
+        <section class="w3">
+          <div class="imgplaceholder"></div>
+          <div class="pro"><h3>Multi-Post Stories</h3></div>
+          <div class="w3p">
+            <p>
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. has been the industry's standard dummy text
+              ever since the 1500s, when an unknown printer took a standard
+              dummy text.
+            </p>
+          </div>
+          <ul class="prolang">
+            <li><span class="css">css</span></li>
+            <li><span class="html">html</span></li>
+            <li><span class="bs">bootstrap</span></li>
+            <li><span class="ruby">Ruby</span></li>
+          </ul>
+          <button type="button" name="See Project">See Project</button>
+        </section>
+        <section class="w4">
+          <div class="imgplaceholder"></div>
+          <div class="pro"><h3>Multi-Post Stories</h3></div>
+          <div class="w4p">
+            <p>
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. has been the industry's standard dummy text
+              ever since the 1500s, when an unknown printer took a standard
+              dummy text.
+            </p>
+          </div>
+          <ul class="prolang">
+            <li><span class="css">css</span></li>
+            <li><span class="html">html</span></li>
+            <li><span class="bs">bootstrap</span></li>
+            <li><span class="ruby">Ruby</span></li>
+          </ul>
+          <button type="button" name="See Project">See Project</button>
+        </section>
+      </section>
+    <footer class="contact-form">
+      <h1>CONTACT US</h1>
+      <p class="contactformtext">
+        If you have an application you are interested in developing, a feature
+        that you need built or a project that needs coding. I’d love to help
+        with it
+      </p>
+      <form action="https://formspree.io/f/mrgrjobb" method="post">
+        <ul class="formlist">
+          <li>
+            <input
+              type="text"
+              id="name"
+              placeholder="Text Input"
+              name="name"
+              maxlength="30"
+              required
+            />
+          </li>
+          <li>
+            <input
+              type="email"
+              id="mail"
+              placeholder="jeremy@gmail.com"
+              name="email"
+              required
+            />
+          </li>
+          <li>
+            <textarea
+              id="msg"
+              placeholder="Enter text here"
+              name="message"
+              maxlength="500"
+            ></textarea>
```

Comment:
- [ ] Please also this textarea it should be required, it's not a good idea to send an empty message.

![llll](https://user-images.githubusercontent.com/61562626/134168186-004b552c-b198-4269-8761-2791ff9d334c.jpeg)

Reply:
@dasileker I have added required to the text field.
May I know what's the optional requirement you want?
As You wrote 
"Since it's a project for mobile and desktop, the view is only a mobile not a desktop, please fix it ."
What should I fix?

Reply:
Hello, @sja-thedude

the optional is optional, but the **textarea** in your form should be required to prevent sending an empty message.

Reply:
Hi, Okay Thanks @dasileker